src/Networking/NexusMods.Networking.Downloaders/DownloadService.cs

```diff
@@ -25,6 +24,9 @@ public class DownloadService : IDownloadService, IDisposable, IHostedService
 {
     /// <inheritdoc />
     public ReadOnlyObservableCollection<IDownloadTask> Downloads => _downloadsCollection;
+    
+    /// <inheritdoc />
+    public AbsolutePath OngoingDownloadsDirectory => _downloadDirectory;
```

Comment:
No need to have a field for this.

```suggestion
    public AbsolutePath OngoingDownloadsDirectory { get; private set; }
```